Components/Form.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+
+function Form() {
+  const [todo, setTodo] = useState("");
+  const [todolist, setTodoList] = useState([]);
+  const [isChecked, setChecked] = useState(false);
+
+  const hanldeChange = (e) => {
+    setTodo(e.target.value);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+
+  const addTodo = () => {
+    //console.log(todo); test
+    if (todo !== "") {
+      const todoInsert = {
+        id: Math.floor(Math.random() * 10000), //Generate random id from 0 to 10000
```

Comment:
For the purpose of this interview that's ok, but ids based on `Math.random` are not a good idea in general.